### PR TITLE
Note about async validate

### PR DESCRIPTION
**Pull request by galvez**

### CODE REVIEW

File: de/api/pages-validate.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ validate({ params, query, store }) {
 }
 ```
 
+You can also return promises:
+
+```js
+validate({ params, query, store }) {
+  return new Promise((resolve) => setTimeout(() => resolve()))
```
Comment (clarkdo) [inline de/api/pages-validate.md]: How about adding a async function？

**Comment by galvez** [comment]
@clarkdo I just realized I edited the `de` file. Fixing.

**Comment by galvez** [comment]
@clarkdo all done.

**Comment by pi0** [comment]
@Atinux Done

**Comment by qm3ster** [comment]
My dudes, shouldn't this be copied to the other "languages" that are currently english? So they stay in sync?

Or even injected into ALL languages?

**Comment by Atinux** [comment]
I think that would be a good idea for translating files easier

**Comment by qm3ster** [comment]
How was your evaluation of GitLocalize? Does it have what it takes to keep the docs properly in sync?
Can we all join there and see the same status, or is it per person?

**Comment by Atinux** [comment]
We will need to find another solution than GitLocalize since the service will be interrupted at the end of this month :/

**Comment by qm3ster** [comment]
How about, all additions must be inserted into all files with a
`<!-- TODO: translate -->`
and all changes must comment out the original and leave a new version to be translated?
Or leave the new version commented, in case we prefer an outdated translation to a regression to english...
So every PR must touch all languages?

**Comment by qm3ster** [comment]
Great, I find out about this great thing only now that it's dead.
And thanks @galvez

**Comment by mbrookes** [comment]
Any more details on the shutdown of gitlocalize? There's no mention on the website, docs site, or blog, and they're still accepting new sign-ups...

**Pull request merged**